sys/include/net/netopt.h

```diff
@@ -269,6 +269,54 @@ typedef enum {
      */
     NETOPT_LAST_ED_LEVEL,
 
+    /** @brief   Radio modulation bandwidth
+     *
+     * Depends on the type of radio used and the regional regulatory and is
+     * used with LoRa/FSK modems.
```

Comment:
This is more confusing then helpful. Which radio expect which type? Is the user supposed to look into the code? Also, if the type depends on the radio (even for LoRa?), how is the top-layer (e.g. a MAC layer) supposed to choose one?